### PR TITLE
Collectives: alliance and collective pallets weights

### PR DESCRIPTION
**Pull request by muharem**
Alliance and collective pallets within Collectives runtime use the weights generated against Substrate dev runtime. 

In this PR we fix it.

### CODE REVIEW

**Comment by muharem** [comment]
/cmd queue -c bench-bot $ pallet collectives-polkadot pallet_collective
/cmd queue -c bench-bot $ pallet collectives-polkadot pallet_alliance

**Comment by muharem** [comment]
/cmd queue -c bench-bot $ pallet collectives-polkadot-dev pallet_collective

**Comment by muharem** [comment]
/cmd queue -c bench-bot $ pallet collectives-polkadot

**Comment by muharem** [comment]
/cmd queue -c bench-bot $ pallet collectives-polkadot collectives pallet_collective

**Comment by muharem** [comment]
/cmd queue -c bench-bot $ pallet collectives-polkadot collectives pallet_alliance

File: parachains/runtimes/collectives/collectives-polkadot/src/weights/pallet_alliance.rs [hunk 1]
```diff
@@ -222,57 +259,75 @@ impl<T: frame_system::Config> pallet_alliance::WeightInfo for WeightInfo<T> {
...
 	// Storage: AllianceMotion Prime (r:0 w:1)
-	// Storage: Alliance UpForKicking (r:0 w:1)
 	fn kick_member() -> Weight {
-		Weight::from_ref_time(46_112_000 as u64)
-			.saturating_add(T::DbWeight::get().reads(6 as u64))
-			.saturating_add(T::DbWeight::get().writes(6 as u64))
+		// Minimum execution time: 120_039 nanoseconds.
+		Weight::from_ref_time(121_205_000 as u64)
+			.saturating_add(T::DbWeight::get().reads(13 as u64))
+			.saturating_add(T::DbWeight::get().writes(8 as u64))
```
Comment (gilescope) [inline parachains/runtimes/collectives/collectives-polkadot/src/weights/pallet_alliance.rs:304]: not that cheap now to kick a member.

Comment (gilescope) [reply parachains/runtimes/collectives/collectives-polkadot/src/weights/pallet_alliance.rs:304]: it's not that expensive in the grand scheme of things though...

**Review by gilescope — APPROVED** [review]

**Review by bkontur — APPROVED** [review]

**Comment by muharem** [comment]
bot merge

**Pull request merged**